src/test/ObjPropsDynamicSetter.Test.Unit/Models/TestClass.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace ObjPropsDynamicSetter.Test.Unit.Models
+{
+    public class TestClass
+    {
+#pragma warning disable CA1051 // Do not declare visible instance fields
+#pragma warning disable IDE1006 // Naming Styles
+#pragma warning disable SA1401 // Fields should be private
+        public object TestField = 1;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/Molnix888/obj-props-dynamic-setter/pullRequest?prid=6241271)